project/osprojects/models.py

```diff
@@ -7,6 +7,7 @@
 from tagging.managers import CustomTaggableManager
 from tagging.models import CustomTag, TaggedItems
 
+
```

Comment:
Note: my linter caused this extra line, but not sure why it's appearing 🤔

Reply:
Apologies for the brain fart and not reviewing this sooner!  I think your linter is enforcing some sort of PEP 8 thing where imports are separated from code by two lines?  Dunno.  Weird.